includes/core/class-wc-payments-customer-service-api.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * Class file for WCPay\Core\WC_Payments_Customer_Service_API.
+ *
+ * @package WooCommerce Payments
+ */
+
+use WCPay\Database_Cache;
+use WCPay\Exceptions\API_Exception;
+use WCPay\Logger;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class handling any customer functionality
+ */
+class WC_Payments_Customer_Service_API {
```

Comment:
Shall we put this class into Core dedicated namespace?

Reply:
Fixed